### ISSUE TITLE
Can variable or directives be the same color as WebStorm ? 

### ISSUE THREAD

**Issue by izosa** [imekachi/webstorm-darcula#2]
[Click to see screenshot example](https://drive.google.com/file/d/1orROh7GTf2rZE1CHu6tuHMiNo3jK8ax0/view?usp=sharing)

**Comment by imekachi** [comment]
@izosa please send me a sample code, I'll take a look.

**Comment by izosa** [comment]
```
<p-card>
    <p-table styleClass="p-datatable-gridlines p-datatable-striped" [columns]="table.columns"
        [value]="list" [paginator]="true" [showCurrentPageReport]="true" [rows]="list.length"
        currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
        [rowsPerPageOptions]="[10,25,50]" (onLazyLoad)="tableLoad($event)" [lazy]="true">

        <ng-template pTemplate="caption">
            Ticket Table
        </ng-template>

        <ng-template pTemplate="header" let-columns>
            <tr>
                <th pSortableColumn="{{column.field}}" *ngFor="let column of table.columns">
                    {{column.header}} <p-sortIcon field="{{column.field}}"></p-sortIcon>
                </th>
            </tr>
        </ng-template>
        <ng-template pTemplate="body" let-item let-columns="table.columns">
            <tr>
                <td *ngFor="let column of table.columns">{{item[column.field]}}</td>
            </tr>
        </ng-template>
    </p-table>

    <ng-template pTemplate="footer">
        <p-button label="Load" icon="pi pi-check" (click)="tableLoad()"></p-button>
    </ng-template>
</p-card>
```

**Comment by imekachi** [comment]
@izosa I've just released [v1.5.1](https://github.com/imekachi/webstorm-darcula/releases/tag/v1.5.1) please update and try it.

![image](https://user-images.githubusercontent.com/5445243/113422445-eea59900-93f6-11eb-9439-d3b4e2f7f437.png)

This is as close as possible to WebStorm.
As you can see that `column` and `$event` are not purple.
In WebStorm, it so smart that it can differentiate local/global variable and highlight global variables as purple italic text.
But in VSCode it doesn't provide that level of granularity for syntax highlighting.

Thank you for using the theme.
If you have other suggestions or having other issues, feel free to leave a comment.
